doto-frontend/src/components/pages/Calendar/Calendar.js

```diff
@@ -0,0 +1,140 @@
+import React, { useState, useEffect } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import Modal from "@material-ui/core/Modal";
+import Backdrop from "@material-ui/core/Backdrop";
+import Fade from "@material-ui/core/Fade";
+import ModalConent from "../../ModalContent";
+import CalendarComponent from "./CalendarComponent";
+import CalendarListView from "./CalendarListView";
+import Fab from "@material-ui/core/Fab";
+import FormatListBulletedIcon from "@material-ui/icons/FormatListBulleted";
+import CalendarTodayIcon from "@material-ui/icons/CalendarToday";
+import AddIcon from "@material-ui/icons/Add";
+import "./Calendar.css";
+
+const useStyles = makeStyles(theme => ({
+    modal: {
+        display: "flex",
+        alignItems: "center",
+        justifyContent: "center",
+    },
+    paper: {
+        backgroundColor: theme.palette.background.paper,
+        border: "2px solid #000",
+        boxShadow: theme.shadows[5],
+        padding: theme.spacing(2, 4, 3),
+    },
+}));
+
+const Calendar = () => {
+    const classes = useStyles();
+    const [listView, setListView] = useState();
+    const [appointments, setAppointments] = useState([]);
+    const [open, setOpen] = React.useState(false);
+
+    const handleOpen = () => {
+        setOpen(true);
+    };
+
+    const handleClose = () => {
+        setOpen(false);
+    };
+
+    useEffect(() => {
+        const tempAppointments = [
+            {
+                title: "Lecture-1",
+                descrition: "Lecture at Auckland Uni",
+                startDate: new Date(2020, 2, 12, 9, 0), // 0 is January
+                endDate: new Date(2020, 2, 12, 11, 0),
+                id: 0,
+                priority: 3,
+                location: "Room 1",
+                color: "blue",
```

Comment:
Based on our add task properties, would we not require a duration field as well?

Reply:
Oh yea, it should have duration. Although, couldn't this be inferred from `endDate - startDate`? is duration supposed to correspond to the dates, or is it separate? @lucycy98 pls help

Reply:
Start date and end date is required by the calendar component we're using, sorry can't use duration. Backend should've configured it to be start date and end date, cos thats what I asked for

Reply:
wdym? Is duration shown on the frontend somwhere?

Reply:
duration as in the field at the modal? or duration as in the duration of the task you see on the calendar

Reply:
> duration as in the field at the modal? or duration as in the duration of the task you see on the calendar

Both? I'm confused now sorry :/

Reply:
In both cases I mentioned, duration is shown on the front end

Reply:
Yea we don't have duration in back end, cause frontend wanted start and end date. Originally I thought we would just have a due date and duration so we can use scheduling to determine a time

Reply:
Like @Dinith1 said, duration can be inferred from the start/end date though right?
> Yea we don't have duration in back end, cause frontend wanted start and end date. Originally I thought we would just have a due date and duration so we can use scheduling to determine a time

Reply:
In the Modal, we do not have a field for Start Date though. So where in the front-end is this being used?

Reply:
the calendar component (the react scheduler), needs start and end date to make an event. thats why i asked back end to explicitly store those 2, and these can be inferred from the duration when the user selects due date and duration